.github/pull-request-template.md

```diff
@@ -0,0 +1,44 @@
+#### What does this PR do?
+A few sentences describing the overall goals of the pull request's commits.
+Why are we making these changes? Is there more work to be done to fully
+achieve these goals?
+
+#### Helpful background context
+
+Describe any additional context beyond what the PR accomplishes if it is likely
+to be useful to a reviewer.
+
+Delete this section if it isn't applicable to the PR.
+
+#### How can a reviewer manually see the effects of these changes?
+
+Explain how to see the proposed changes in the application if possible.
+
+Delete this section if it isn't applicable to the PR.
+
+#### What are the relevant tickets?
+
+- https://mitlibraries.atlassian.net/browse/DIP-
```

Comment:
Not a big deal, but I'd remove the `DIP` bit here and replace with the project slug this repo typically works out of... if there isn't a typical project in Jira for this, removing the starter may be best.